src/client/algod.js

```diff
@@ -34,6 +34,18 @@ function Algod(token, baseServer = "http://r2.algorand.network", port = 4180) {
         return res.body;
     };
 
+    /**
+     * Takes an object and convert its note field to Buffer, if exist.
+     * @param o
+     * @returns {*}
+     */
+    function noteb64ToNote(o) {
```

Comment:
Can we move this function to the top of the file?

Reply:
Moved to the top of the file.